after/queries/typescript/highlights.scm

```diff
@@ -0,0 +1,12 @@
+;; extends
+[
+("=>" @operator.fat_arrow (#set! "priority" 1000))
+]
+
+[
+(enum_declaration (enum_body (property_identifier) @property.enum) (#set! "priority" 1000))
+]
```

Comment:
These are already marked `@constant`, do you want them highlighted differently if in enums specifically?

```suggestion
(enum_body
  (property_identifier) @enum.member)
```

Reply:
So interestingly for your suggested query (which is much nicer than mine) I did have to apply `(#set! "priority" 1000)` as `@property.typesript` takes priority here and in theme, I wanted enums to be highlighted differently.

Reply:
Moving it lower in the queries file will give it priority FYI, and default priority is 100 so 105 or so would work **if you really need it** and shuffling queries wouldn't work